### PR TITLE
Fix broken Utils::tryFopen on PHP 8

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: src/Utils.php [hunk 1]
```diff
@@ -358,14 +358,24 @@ public static function tryFopen($filename, $mode)
...
                 $filename,
                 $mode,
                 func_get_args()[1]
             ));
         });
 
-        $handle = fopen($filename, $mode);
+        try {
+            $handle = fopen($filename, $mode);
+        } catch (\Throwable $e) {
```
Comment (Nyholm) [inline src/Utils.php:370]: This will not work on PHP 5.4. Right?

Comment (GrahamCampbell) [reply src/Utils.php:370]: PHP 5 only ever raises errors, and never "throws" anything. :)

Comment (Nyholm) [reply src/Utils.php:370]: Yeah, but does `Throwable` exist in PHP 5.4?

Comment (GrahamCampbell) [reply src/Utils.php:370]: That's fine. Classes in a catch block don't trigger the autoloader.

Comment (Nyholm) [reply src/Utils.php:370]: True, But we will never catch an exception in PHP 5.4. Right?

Comment (GrahamCampbell) [reply src/Utils.php:370]: Yes. That's no problem because:

> `fopen` on PHP 5 only ever raises errors, and never "throws" anything.

Comment (Nyholm) [reply src/Utils.php:370]: Oh, sorry. I understand now. 

Thank you.

**Review by Nyholm — APPROVED** [review]

**Pull request merged**